server/routes/bitbucket/bitbucket.py

```
@@ -122,7 +123,7 @@ def bitbucket_callback():
                 frontend_url=FRONTEND_URL,
             )
 
-        logger.info(f"Received Bitbucket code: {code[:5]}...")
+        logger.info(f"Received Bitbucket code: {sanitize(code)[:5]}...")
```

Ruff (0.15.11)

[warning] Logging statement uses f-string

(G004)